CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [15.0.0] - 2023-02-01
```

Comment:
sorry I already merged it, but could you fix the version to 16.0.1?

Reply:
https://github.com/WinterFramework/winter/pull/262 here is a new pull request with changelog fix